### ISSUE TITLE
Updating new code to other entities after uploading to one already

### ISSUE THREAD

**Issue by Jazzelhawk** [INPStarfall/Starfall#279]
Basically a screen will run old code when updating it with new code that has already been uploaded to another screen.
To reproduce this, spawn two screens, update the first screen with some new code, then update the second screen with the same code and the second screen will run with the old code.

**Comment by Xandaros** [comment]
@awilliamson Didn't you fix this?

**Comment by awilliamson** [comment]
@Xandaros I remember investigating this with @Jazzelhawk on INP. Pretty sure we found the root cause, I believe the changes are on Jazzel's branch waiting to come in.

**Comment by Jazzelhawk** [comment]
Nope, I found the cause but you were going to totally redo the code sending stuff. @awilliamson made [this branch](https://github.com/awilliamson/Starfall/commit/91961042419c790170dcb081dacd448938935ebe).
But I think the cause was to do with the `local hashes = {}` in `lua/entities/starfall_screen/cl_init.lua` being shared between each screen. I suggested creating the hashes table on the entity which is a simple fix but you guys said you wanted it on the instance.

**Comment by awilliamson** [comment]
@Jazzelhawk I thought the changes on that branch worked? I managed to repeat the bug from your steps, tried the hashbrowns branch, and the problem is gone? 'Should' work as a fix?
